cluster/ubuntu/util.sh

```diff
@@ -188,8 +188,8 @@ function create-etcd-opts() {
   cat <<EOF > ~/kube/default/etcd
 ETCD_OPTS="\
  -name infra\
- -listen-client-urls http://0.0.0.0:4001\
- -advertise-client-urls http://127.0.0.1:4001"
+ -listen-client-urls http://127.0.0.1:4001,http://${1}:4001\
```

Comment:
yeah, i mean all of the clients which want to talk to the etcd locates locally as with etcd cluster in my use case